### ISSUE TITLE
No support for PHP 5.6 splat operator

### ISSUE THREAD

**Issue by bashofmann** [rtens/mockster#16]
When mocking a method that uses the 5.6 splat operator the following strict error is triggered:

```
E_STRICT: Declaration of Mock_ClassName_NoConstructor::methodName() should be compatible with ClassName::methodName(...$argList)
```

**Comment by rtens** [comment]
Thanks for reporting. Guess I need to bump supported PHP versions up a number. Looks like the parameter is not recognized at all.

The interpreter is actually a little too strict for my taste when it comes to compatibility of method signatures. This has more often been a burden for me than a help. Especially when it comes to (non-existing) type inference.

Anyway. I'll fix this when I introduce support for PHP 5.6. I suppose this is not a blocker for you. Let me know if it is.

**Comment by bashofmann** [comment]
No in this case it is not a blocker, because we can just not mock this method
